crates/subspace-networking/src/node_runner.rs

```diff
@@ -523,6 +557,15 @@ impl NodeRunner {
                     IfDisconnected::TryConnect,
                 );
             }
+            Command::CheckConnectedPeers { result_sender } => {
+                let connected_peers_present = self.swarm.connected_peers().next().is_some();
```

Comment:
FYI, there is `connected_peers_count` already in `Shared`, might be faster to access comparing to accessing concrete peers that you don't care about anyway.